zerver/views/message_send.py

```diff
@@ -285,6 +286,7 @@ def send_message_backend(
 
     if (delivery_type == "send_later" or delivery_type == "remind") and defer_until is not None:
         return handle_deferred_message(
+            request,
```

Comment:
Probably same with this one; the view helper could return `str(deliver_at_usertz)` and then this could do the `json_success` part.